### PR TITLE
Allow `designatedTotal` Input for Pie Grid

### PR DESCRIPTION
**Pull request by brettnew**
**What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
Currently the pie grid chart assumes that each value in the series is part of the total amount/ percentage. For example, if Germany has a value of 750k, and the US has a value of 250k, the pie grid chart would display Germany as 75%, and the US as 25%. This works if you only want to compare these two countries to each other.

**What is the new behavior?**
However, if you want to compare these countries' values to the entire world's values, there is no way to currently do that. With this PR, you can set the `designatedTotal`, which will be the number that each country's value will be compared to. For example, if the `designatedTotal` is set to 5 million, Germany's percentage will now be 15%, and the US will be 5%.

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

Demo without `designatedTotal`:
image

Demo with `designatedTotal` set to 49,737:
image

### CODE REVIEW

File: src/common/base-chart.component.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ export class BaseChartComponent implements OnChanges, AfterViewInit, OnDestroy {
 
   @Output() select = new EventEmitter();
 
+  designatedTotal: number;
```
Comment (marjan-georgiev) [inline src/common/base-chart.component.ts]: This doesn't look like it belongs to the base chart, as it it not used by all charts of the library. Perhaps move it to the number card and pie grid components?

Comment (brettnew) [reply src/common/base-chart.component.ts]: Ok, good point; I've moved to the number card component.

File: src/number-card/number-card.component.ts [hunk 2]
```diff
@@ -48,6 +48,7 @@ export class NumberCardComponent extends BaseChartComponent {
   @Input() valueFormatting: any;
   @Input() labelFormatting: any;
 
+  designatedTotal: number;
```
Comment (marjan-georgiev) [inline src/number-card/number-card.component.ts]: It also needs to be decorated as an `@Input()`

File: release/pie-chart/pie-grid.component.d.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { ViewDimensions } from '../common/view-dimensions.helper';
 import { ColorHelper } from '../common/color.helper';
 import { BaseChartComponent } from '../common/base-chart.component';
 export declare class PieGridComponent extends BaseChartComponent {
+    designatedTotal: number;
```
Comment (marjan-georgiev) [inline release/pie-chart/pie-grid.component.d.ts]: Let's remove the change from the release files please. They are autogenerated during release, and should not be committed to the pull request.

**Comment by marjan-georgiev** [comment]
Thanks! 👍

**Comment by brettnew** [comment]
Thanks for the feedback!

**Comment by jvincilione** [comment]
@marjan-georgiev  When will this be released? Looks like it's already been added to the docs, so I was trying to use it today, and it looks like it has not been released.

**Comment by marjan-georgiev** [comment]
Released today in 6.1.0

**Comment by digitalore** [comment]
Will _designatedTotal_ be added as an input for "ngx-charts-advanced-pie-chart" as well?

**Comment by noman115pk** [comment]
@marjan-georgiev @brettnew  Can we show this designatedTotal as a tool tip while hovering over unfilled area? Only numerators are shown as Total in figures. How can we show denominators as Expected or total amount?
